server/game/core/Player.js

```diff
@@ -288,16 +288,33 @@ class Player extends GameObject {
     }
 
     /**
-     * Returns if a card is in play (unots, upgrades, provinces, holdings) that has the passed trait
+     * Returns if a card is in play (units, upgrades, provinces, holdings) that has the passed trait
```

Comment:
This is a mistake in the docs that I missed - since these methods currently only return units, can we update the docstr for this the surrounding methods to say that